paasta_tools/paasta_metastatus.py

```diff
@@ -286,6 +286,26 @@ def print_results_for_healthchecks(summary, ok, results, verbose):
             print_with_indent(line, 2)
 
 
+def print_extra_mesos_data():
+    mesos_state = get_mesos_state_from_leader()
```

Comment:
Do we have to get the state again? I thought we had it already.

If not can you inject this data?